src/pages/applications.js

```diff
@@ -295,6 +296,23 @@ const emergingProjects = [
       {label: 'Website', url: 'https://www.loxilb.io/'},
     ],
   },
+  {
+    logoUrl: 'https://github.com/pyroscope-io/pyroscope',
+    name: 'Pyroscope',
+    logo: pyroscopeLogo,
+    title: 'Continuous Profiling Platform',
+    description: `Pyroscope is an open source project centered around
+    continuous profiling, particularly in a Kubernetes context.
+    It leverages eBPF along with a custom storage engine
```

Comment:
small nit given you mentioned it in your PR description, but could you state it here, e.g. `It leverages eBPF as its core technology along with [...]`? Thanks

Reply:
updated it :+1: